docs/docs/rest-api/public/api/v2/apps.raml

```diff
@@ -0,0 +1,416 @@
+
+###
+# Get a list of applications
+#
+get:
+  description:
+    Get the list of running applications. Several filters can be applied via the following query parameters.
+  is: [ secured ]
+  queryParameters:
+    cmd:
+      required: false
+      description: Filter the result to only return apps whose `cmd` field contains the given value
+      example: java
+    id:
+      required: false
+      description: Filter the result to only return apps whose `id` is or contains the given value
+      example: /us-east/database/memsql
+    label:
+      required: false
+      description:
+        A label selector query contains one or more label selectors, which are comma separated.
+        Marathon supports three types of selectors existence-based, equality-based and set-based.
+        In the case of multiple selectors, all must be satisfied so comma separator acts as an AND logical operator.
+        Labels and values must consist of alphanumeric characters plus `-` `_` and `.` `-A-Za-z0-9_.`.
+        Any other character is possible, but must be escaped with a backslash character.
+
+        * <code>Existence based Selector Query</code> Matches the existence of a label <br/>
+          Example&#58; my_label,environment
+
+        * <code>Equality based Selector Query</code> Matches existence of labels and the (non) equality of the value.<br/>
+          Example&#58; environment==production, tier!=frontend
+
+        * <code>Set based Selector Query</code> Matches existence of labels and the (non) existence of the value in a given set<br/>
+          Example&#58; environment in (stage,production), tier notin (frontend, service)
+
+      example: my_label, environment==production, tier!=frontend\ tier, deployed in (us, eu), deployed notin (aa, bb)
+    embed:
+      required: false
+      description:
+        Embeds nested resources that match the supplied path.
+        You can specify this parameter multiple times with different values.
+
+        - <code>apps.tasks</code> embed all tasks of each application<br/>
+        Note&#58; if this embed is definded, it automatically sets <code>apps.deployments</code> but this will change in a future release.
+        Please define all embeds explicit.
```

Comment:
Good idea to add that hint. However, please `s/explicit/explicitly`, as this is an adverb to `define`. This does also hold for the 2 occurrences below.